views/play.erb

```diff
@@ -0,0 +1,16 @@
+<img src="images/rockpaperscissors.jpg" alt="rock paper scissors images" width="450" height="200">
+
+<h1>Hello, <%= @player_name %>!</h1>
+<h2>Please choose your move:</h2>
+
+<form action="move_rock" method="post">
```

Comment:
You could have this in a single form with multiple options, where the route would be to (something like) /player_move and each button would send a different value (i.e. rock, paper or scissors). Then, you could save the player move into the `move` variable, and just have a single view for the player move which changes depending on what the player move was. (hope this makes sense, happy to talk this through if not)